scripts/o3de/o3de/engine_template.py

```diff
@@ -2300,6 +2299,71 @@ def create_gem(gem_path: pathlib.Path,
     # Register the gem with the either o3de_manifest.json, engine.json or project.json based on the gem path
     return register.register(gem_path=gem_path) if not no_register else 0
 
+def create_repo(repo_path: pathlib.Path,
+                repo_uri: str,
```

Comment:
What's the difference between repro_uri and origin_url? Origin is like the "author" right? Could we add a comment after that param?

Reply:
The comment is in the help menu, its the original author website